fabric-data-generation-api-v1/src/testmod/java/net/fabricmc/fabric/test/datagen/DataGeneratorTestEntrypoint.java

```diff
@@ -111,6 +138,25 @@ protected void generateTags() {
 		}
 	}
 
+	private static class TestBiomeTagsProvider extends FabricTagProvider.DynamicRegistryTagProvider<Biome> {
+		private TestBiomeTagsProvider(FabricDataGenerator dataGenerator) {
+			super(dataGenerator, Registry.BIOME_KEY, "biomes", "Biome Tags");
+		}
+
+		@Override
+		protected void generateTags() {
+			FabricTagBuilder<Biome> builder = getOrCreateTagBuilder(TagFactory.BIOME.create(new Identifier(MOD_ID, "biome_tag_test")))
+					.add(BiomeKeys.BADLANDS.getValue())
+					.add(BiomeKeys.BASALT_DELTAS.getValue());
```

Comment:
It could be nice to also have `RegistryKey` overloads so you wouldn't have to do the `getValue` each time.